### ISSUE TITLE
Does not work for Google Docs in Firefox (v. 90.0.1)

### ISSUE THREAD

**Issue by jooyoungseo** [nvaccess/nvda#12671]
### Steps to reproduce:

* Open any Google Docs in Firefox (version 90.0.1).

### Actual behavior:

* NVDA does not read any content exclusively in Google Docs. This means NVDA is working well in other web pages.

### Expected behavior:

* Google Docs content should be read.

### System configuration

* Screen reader support is enabled in Google Docs.

* Braille support is enabled in Google Docs.

#### NVDA installed/portable/running from source:

* Installed version.

#### NVDA version:

* 2021.1

#### Windows version:

* Microsoft® Windows® Operating System Version 10.0.19041.1

#### Name and version of other software in use when reproducing the issue:

* Firefox Version 90.0.1

#### Other information about your system:

* Latest MathPlayer is installed.

### Other questions

* None.

#### Does the issue still occur after restarting your computer?

* Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

* No.

#### If add-ons are disabled, is your problem still occurring?

* Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

* Yes.

**Comment by feerrenrut** [comment]
I'm unable to reproduce this issue. Could you provide a debug level log please?
[How to collect a log](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps)

**Comment by Adriani90** [comment]
I cannot reproduce either, and we didn't get further updates from the author since many years. Closing as works for me.
